### DISCUSSION TITLE
It would be great if crow translate and olive editor are added.

### DISCUSSION THREAD

**Discussion by ahmetcanisik** [ChrisTitusTech/winutil#2444] [Ideas]
Hello, I have been using winutil for a long time and I would like to inform you that I would be pleased to see these applications in it.

- [crow translate](https://apps.kde.org/crowtranslate/) (translator program)
- [olive editor](https://github.com/olive-editor/olive) (open source video editing program)

**Comment by MyDrift-user** [comment]
Hey @ahmetcanisik 

Crow translate seems ro be a linux exclusive application. I was not able to find a windows version of it.

**Reply by ahmetcanisik** [reply]
The project has been moved under the kde umbrella. Here is the download page available as of v3.0.0. You can reach the download page [here](https://download.kde.org/stable/crow-translate/).

**Comment by ahmetcanisik** [comment]
Yes, I'm sorry for making that suggestion. If you want, you can look at this link. Here you will see the old archived [github repository](https://github.com/crow-translate/crow-translate). Olive Editor also works quite well, although it is still in its early stages.
